app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -59,7 +59,7 @@ private void nextActivity() {
             LOGGER.info("Sending my profile with name: " + myProfile.getName() + ", public key: "
                     + myProfile.getPublicKey().toString());
             final Contact myProfileAsContact = new Contact(myProfile.getName(), new SimpleKey("simplekey", "73890ien"));
-            this.serviceLocator.getBluetoothHandler().send(myProfileAsContact);
+            this.serviceLocator.getBluetoothHandler().send(this.serviceLocator.getBluetoothHandler().objectToBytes(myProfileAsContact));
```

Comment:
> One send method, namely send(byte[]) should be enough indeed.

😕 Thats not what I meant... by "or even one, not sure if we really need two" I meant that I'm not sure if we still needed the `send(byte[])` version because NFC just needs the bytes and doesn't send anything through the handler.


> I don't think it's a problem if we have two methods, but I don't have a problem with one method either.

I do have a problem with it, that is why I started this thread 😅


> If you look at my latest changes I went from 1 to 2 methods, actually this was there since @ericcornelissen posted his 2nd comment :p

I would actually like to see just the other method, so `send(Object)` ([looking at the diff](https://github.com/ericcornelissen/NervousFish/pull/231/files/2058483bfce4465c711e98be6fb407e4655b5248#diff-12207c280a7c5d460c92841b36106522R62) you're now using `send(byte[])`, right?)